### PR TITLE
Int 7426

### PR DESCRIPTION
**Pull request by sl45082**
# Description

Updating the documentation to help with integration adoption.

## Summary

Updates to comments in env.example and README.md

## Type of change

No breaking changes.

### CODE REVIEW

**Review by Gonzalo-Avalos-Ribas — COMMENTED** [review]
This is awesome, thanks! 😊 
Would love if we could add more information on how to add a new user to the Jira account so that they can use the integration.
What i mean is that you assume that someone *has* an account, and i'd like to see some instructions on how to invite someone that has no account.
The questions that maybe arise are: How do we add someone new? How do we give him permissions to execute the integration? Do we have to give him permissions to create new entities (such as admin of the account? How?

**Review by gastonyelmini — DISMISSED** [review]
Looks good to me!

**Review by mishelashala — DISMISSED** [review]

**Review by gastonyelmini — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,9 +27,11 @@ with JupiterOne in the [integration documentation](docs/jupiterone.md).
    from Node's `process.env`, loaded from `.env`. That file has been added to
    `.gitignore` to avoid commiting credentials.
 
+## Development
+
 ### Running the integration
 
-1. `yarn start` to collect data
+1. `yarn start` to run integrtion and collect data
```
[the remark this answers is not in the document]
Comment (sl45082) [reply README.md:34]: will fix and update.

**Review by gastonyelmini — APPROVED** [review]
Looks good!

**Review by Gonzalo-Avalos-Ribas — APPROVED** [review]
💯

**Review by gastonyelmini — APPROVED** [review]

**Pull request merged**